pom.xml

```diff
@@ -155,5 +155,52 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>format</id>
+            <activation>
+                <activeByDefault>true</activeByDefault>
+                <property>
+                    <name>!no-format</name>
+                </property>
+            </activation>
```

Comment:
Which of these options will take precedence? I understand, that the desired behavior is not to run the check when `no-format` property is defined. Will this override the `activeByDefault` setting? Also, isn't this redundant, when you just can run `mvn clean test -P '!format'` [1]?

[1] http://maven.apache.org/guides/introduction/introduction-to-profiles.html#Deactivating_a_profile

Reply:
good point, it was taken from Quarkus, I will update the PR